src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -1243,6 +1243,34 @@ static bool convertArticulationToSymId(const QString& mxmlName, SymId& id)
     }
 }
 
+//---------------------------------------------------------
+//   convertFermataToSymId
+//---------------------------------------------------------
+
+/**
+ Convert a MusicXML fermata name to a MuseScore fermata.
+ */
+
+static SymId convertFermataToSymId(const QString& mxmlName)
+{
+    QMap<QString, SymId> map; // map MusicXML fermata name to MuseScore symbol
+    map["normal"]           = SymId::fermataAbove;
+    map["angled"]           = SymId::fermataShortAbove;
+    map["square"]           = SymId::fermataLongAbove;
+    map["double-angled"]    = SymId::fermataVeryShortAbove;
+    map["double-square"]    = SymId::fermataVeryLongAbove;
+    map["double-dot"]       = SymId::fermataLongHenzeAbove;
+    map["half-curve"]       = SymId::fermataShortHenzeAbove;
+    map["curlew"]           = SymId::curlewSign;
```

Comment:
Maybe like this?
```suggestion
    // map MusicXML fermata name to MuseScore symbol
    static const QMap<QString, SymId> map {
        { "normal",        SymId::fermataAbove },
        { "angled",        SymId::fermataShortAbove },
        { "square",        SymId::fermataLongAbove },
        { "double-angled", SymId::fermataVeryShortAbove },
        { "double-square", SymId::fermataVeryLongAbove },
        { "double-dot",    SymId::fermataLongHenzeAbove },
        { "half-curve",    SymId::fermataShortHenzeAbove },
        { "curlew",        SymId::curlewSign }
    };
```

Reply:
Of course possible, but quite different from the artuculations above

Reply:
Yes, there's enough left to do for another day.